apisix/discovery/nacos/init.lua

```diff
@@ -281,11 +290,18 @@ local function fetch_full_registry(premature)
                 up_apps[namespace_id]
                     [group_name][service_info.service_name] = nodes
             end
-            core.table.insert(nodes, {
+
+            local node = {
                 host = host.ip,
                 port = host.port,
                 weight = host.weight or default_weight,
-            })
+            }
+
+            if is_grpc(scheme) and host.metadata and host.metadata.gRPC_port then
```

Comment:
@vcqr 
Before we can merge it, could you add a link to the doc about the usage of gRPC_port?